src/components/report-problem-issue-type/report-problem-issue-type.container.js

```diff
@@ -2,13 +2,14 @@
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import ReportProblemIssueType from './report-problem-issue-type.component';
+import { cacheLastAnnotation } from '../../actions/trail-annotations.action-creator';
 
-const mapStateToProps = ({userLocation}) => {
-  return { userLocation };
+const mapStateToProps = ({photo, annotations}) => {
+  return { photo, annotations };
 };
 
 const mapDispatchToProps = (dispatch) => {
-  return bindActionCreators({
+  return bindActionCreators({cacheLastAnnotation
```

Comment:
what is cacheLastAnnotation do? I know its an action creator but what is it for?